book/service_container.rst

```diff
@@ -812,6 +817,64 @@ allow for an optional dependency::
             // ...
         }
 
+The "ignore" behavior when a service does not exist is context sensitive:
+
+- When used as a standard argument, the argument will be set to ``null``;
+- When used within a method call, the method call itself will be removed;
+- When used on a collection argument, the argument will be removed.
```

Comment:
Yeah, I think we should omit it here and add it after merging your PR up to `master`.